packages/voorbeeld-design-tokens/documentation/number-badge/utrecht.numberBadge.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { BadgeCounter } from '@utrecht/component-library-react/dist/css-module';
+
+const meta = {
+  id: 'utrecht-badge-counter-group',
+  title: 'Components/Number Badge/Utrecht',
+  component: BadgeCounter,
+  render: () => (
+    <>
+      <BadgeCounter>9</BadgeCounter>
+    </>
+  ),
+} satisfies Meta<typeof BadgeCounter>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+export const VoorbeeldTheme: Story = {
+  parameters: {
+    theme: 'voorbeeld-theme',
+    title: 'Voorbeeld theme',
```

Comment:
Deze `title` in `parameters` moet `name` worden onder de story zelf, ook op regel 27